src/stravalib/model.py

```diff
@@ -952,6 +976,7 @@ class Segment(
         "elevation_high": uh.meters,
         "elevation_low": uh.meters,
         "total_elevation_gain": uh.meters,
+        "activity_type": enum_value,
```

Comment:
This tiny change is there to give the segment's `activity_type` the same behavior as the activity's `type` (i.e., returning the string representation of the nested type). This is for backward compatibility with the legacy model.